upgrade.sh

```diff
@@ -25,17 +25,17 @@ case $step in
     done
     sudo systemctl stop nabd.socket || echo -n ""
     sudo systemctl stop nabd.service || echo -n ""
-  
+
     sudo -u ${owner} touch /tmp/pynab.upgrade
     sudo chown ${owner} /tmp/pynab.upgrade
     echo "Updating code - 1/?" > /tmp/pynab.upgrade
     cd ${root_dir}
     if [[ $EUID -ne ${ownerid} ]]; then
-      sudo -u ${owner} git pull
+      sudo -u ${owner} git pull --no-rebase
```

Comment:
I don't understand why we would want --no-rebase here.

Reply:
The context is that when there is a default rebase strategy active in the global git config, the up-to-date message returned by Git is slightly different from the one tested (see PR discussion earlier). I initially proposed to adjust the message matching to support "Already up-to-date" and "Branch XXX is up-to-date". @f-laurens suggested that the message could be made consistent by using --no-rebase.